.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Integration Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - production
```

Comment:
Shouldn't this be `prod`?

Reply:
beat me by seconds: https://github.com/brave-intl/challenge-bypass-server/pull/829